src/DotLiquid/Properties/AssemblyInfo.cs

```diff
@@ -25,8 +25,8 @@
 
 [assembly: Guid("75d1dc49-0097-4f60-b5fd-0e43d213b38b")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
-[assembly: AssemblyInformationalVersion("1.0.0.0")]
+[assembly: AssemblyVersion("2.0.315.0")]

```

Comment:
Probably the 1.0.0.0 version is a placeholder for build to overwrite with actual build value. Eitherway, the assembly build version should not be baked in the code

Reply:
Yes, same reason as the first comment. I was flying solo, but I gladly will move to DotLiquid again when my original fix (or a similar one) is accepted. I can go back and make a new PR if you're in the project and willing to accept it. Thanks.